package.json

```diff
@@ -32,7 +32,8 @@
   "dependencies": {
     "deep-equal": "^1.0.1",
     "object-assign": "^4.1.1",
-    "react-side-effect": "^1.1.0"
+    "react-side-effect": "^1.1.0",
+    "prop-types": "^15.5.4"
```

Comment:
_(sorry for nitpicking)_
I guess this line should be moved before react-side-effect (to keep the list sorted alphabetically).